config/v3_0/types/mode.go

```diff
@@ -24,3 +26,13 @@ func validateMode(m *int) error {
 	}
 	return nil
 }
+
+func validatePermissionBits(m *int) error {
+	if m != nil {
+		mode := uint32(int(*m))
+		if mode&syscall.S_ISUID != 0 || mode&syscall.S_ISGID != 0 || mode&syscall.S_ISVTX != 0 {
```

Comment:
This is okay, but I think it's also okay to write `mode & 07000 != 0`.  The definitions of those bits are well-known.

Reply:
ok, so you're suggesting this?
```suggestion
		if mode& 07000 !=0 {
```

Reply:
Right.